### PR TITLE
Add new help commands and change some of the old ones

### PR DESCRIPTION
**Pull request by AustinXII**
This is intended to help new people who are running a server and don't know what the admins commands do, I stayed away from the /eval commands seeing that the command can be risky and it would probably be best if they didn't know what it does.

### CODE REVIEW

File: commands.js [hunk 1]
```diff
@@ -1480,6 +1480,7 @@ var commands = exports.commands = {
 		fs.writeFile('data/learnsets.js', 'exports.BattleLearnsets = ' + JSON.stringify(Tools.data.Learnsets) + ";\n");
 		this.sendReply("learnsets.js saved.");
 	},
+	savelearnsetshelp: ["Changes the moveset of a pokémon. Requires: ~"]
```
Comment (kubetz) [inline commands.js]: `,` seems to be missing here.

File: commands.js [hunk 2]
```diff
@@ -1701,6 +1706,7 @@ var commands = exports.commands = {
 			});
 		});
 	},
+	updateserverhelp: ["/updateserver - updates the server to a newer version. Requires: Console Access"],
```
Comment (Bryan-0) [inline commands.js]: 'updateserverhelp: ["/updateserver - updates the server to a newer version. Requires: Console Access"],' Capital letter is missing on "Updates".

**Comment by Bryan-0** [comment]
Also, check Travis CI.

File: commands.js [hunk 3]
```diff
@@ -1588,6 +1591,7 @@ var commands = exports.commands = {
 
 		this.logEntry(user.name + " used /endemergency");
 	},
+	emergencyhelp: ["/emergency - Puts the server into energency mode, making some features limited until a fix. Requires: ~"],
```
Comment (kubetz) [inline commands.js]: There is a typo in `energency mode`.

File: commands.js [hunk 4]
```diff
@@ -1715,6 +1721,7 @@ var commands = exports.commands = {
 		}
 		this.logEntry(user.name + " used /crashfixed");
 	},
+	crashfixedhelp: ["/crashfixed - Annoucnes that the server has been fixed without the need of a restart. Requires: ~"],
```
Comment (kubetz) [inline commands.js]: There is a typo in `Annoucnes`.

File: commands.js [hunk 5]
```diff
@@ -1490,6 +1491,7 @@ var commands = exports.commands = {
 		this.logModCommand("The ladder was disabled by " + user.name + ".");
 		this.add("|raw|<div class=\"broadcast-red\"><b>Due to high server load, the ladder has been temporarily disabled</b><br />Rated games will no longer update the ladder. It will be back momentarily.</div>");
 	},
+	disableladderhelp: ["Disables the ladder, which will stop rated battles from being played and stop the ladder from updating. Requires: ~"],
```
Comment (Bryan-0) [inline commands.js]: Why not: "disableladderhelp: ["*/disableladder -* Disables the ladder, which will stop rated battles from being played and stop the ladder from updating. Requires: ~"],"

File: commands.js [hunk 6]
```diff
@@ -1500,6 +1502,7 @@ var commands = exports.commands = {
 		this.logModCommand("The ladder was enabled by " + user.name + ".");
 		this.add("|raw|<div class=\"broadcast-green\"><b>The ladder is now back.</b><br />Rated games will update the ladder now.</div>");
 	},
+	enableladderhelp: ["Enables the ladder, which will allow rated battles to commence and allows the ladder to update. Requires: ~"],
```
Comment (Bryan-0) [inline commands.js]: Here too: "enableladderhelp: ["*/enableladder* - Enables the ladder, which will allow rated battles to commence and allows the ladder to update. Requires: ~"],"

**Comment by AustinXII** [comment]
Sorry I had to rush to finish this because my next block in school was about to start

File: commands.js [hunk 7]
```diff
@@ -1490,6 +1491,7 @@ var commands = exports.commands = {
 		this.logModCommand("The ladder was disabled by " + user.name + ".");
 		this.add("|raw|<div class=\"broadcast-red\"><b>Due to high server load, the ladder has been temporarily disabled</b><br />Rated games will no longer update the ladder. It will be back momentarily.</div>");
 	},
+	disableladderhelp: ["/disableladder - Disables the ladder, which will stop rated battles from being played and stop the ladder from updating. Requires: ~"],
```
Comment (Bryan-0) [inline commands.js]: Leaders (&) can disable ladder too, not just Admins (~).

File: commands.js [hunk 8]
```diff
@@ -1650,6 +1654,7 @@ var commands = exports.commands = {
 		Rooms.global.send('|refresh|');
 		this.logEntry(user.name + " used /refreshpage");
 	},
+	refreshpagehelp: ["/refreshpage - Forcefully refreshes the page for all users on the server. Requires: ~"],
```
Comment (Slayer95) [inline commands.js]: This isn't quite the case. It only works for users in lobby.

File: commands.js [hunk 9]
```diff
@@ -1779,6 +1786,7 @@ var commands = exports.commands = {
 		}
 		return;
 	},
+	memoryusagehelp: ["/memoryusage - Shows the amount of memory the server is using. Requires: Console Access"],
```
Comment (Slayer95) [inline commands.js]: This could just say `~`

File: commands.js [hunk 10]
```diff
@@ -1701,6 +1706,7 @@ var commands = exports.commands = {
 			});
 		});
 	},
+	updateserverhelp: ["/updateserver - Updates the server to a newer version. Requires: Console Access"],
```
Comment (Slayer95) [inline commands.js]: This doesn't really say anything...

File: commands.js [hunk 11]
```diff
@@ -1715,6 +1721,7 @@ var commands = exports.commands = {
 		}
 		this.logEntry(user.name + " used /crashfixed");
 	},
+	crashfixedhelp: ["/crashfixed - Announces that the server has been fixed without the need of a restart. Requires: ~"],
```
Comment (Slayer95) [inline commands.js]: This doesn't either :|

File: commands.js [hunk 12]
```diff
@@ -1588,6 +1591,7 @@ var commands = exports.commands = {
 
 		this.logEntry(user.name + " used /endemergency");
 	},
+	emergencyhelp: ["/emergency - Puts the server into emergency mode, making some features limited until a fix. Requires: ~"],
```
Comment (Slayer95) [inline commands.js]: `emergency` is a (mostly) deprecated function. We shouldn't document it imo, though maybe others are of different opinion?

File: commands.js [hunk 13]
```diff
@@ -1480,6 +1480,7 @@ var commands = exports.commands = {
 		fs.writeFile('data/learnsets.js', 'exports.BattleLearnsets = ' + JSON.stringify(Tools.data.Learnsets) + ";\n");
 		this.sendReply("learnsets.js saved.");
 	},
+	savelearnsetshelp: ["/savelearnsets - Changes the moveset of a pokémon. Requires: ~"],
```
Comment (Slayer95) [inline commands.js]: Totally misleading. Also deprecated. See comment below.

File: commands.js (hunk 12 above)
Comment (Zarel) [inline commands.js]: I'm pretty sure it does exactly nothing right now...

File: commands.js (hunk 9 above)
Comment (Zarel) [inline commands.js]: Does this command even do anything? iirc, the last time I tried it, it froze PS. Maybe we should just remove it.

File: commands.js (hunk 11 above)
Comment (Zarel) [inline commands.js]: It's also inaccurate - the main use of `/crashfixed` is to end a server lockdown caused by a crash.

File: commands.js (hunk 9 above)
Comment (Slayer95) [inline commands.js]: It should be simplified to just returning the contents of `process.memoryUsage()`

File: commands.js (hunk 10 above)
Comment (Zarel) [inline commands.js]: `hotpatch` isn't `console`. All admins have `hotpatch` access, not just console admins.

Comment (Zarel) [inline commands.js]: And yes, this documentation is useless unless it explains exactly how it updates the server.

File: commands.js (hunk 13 above)
Comment (Zarel) [inline commands.js]: I'd say "wrong", not just misleading. Also deprecated, apparently.

File: commands.js (hunk 7 above)
Comment (Zarel) [inline commands.js]: I would recommend not documenting this command at all. It's mainly a server load command and doesn't have use outside of PS Main.

File: commands.js (hunk 8 above)
Comment (Zarel) [inline commands.js]: Rooms.global.send('|refresh|');

Hm, if it only works for uses in lobby, that wasn't its intent. Maybe we should keep track?

Comment (Slayer95) [inline commands.js]: Yea, that's probably a leftover from before rooms were implemented.

**Comment by Zarel** [comment]
You know how to squash, right?

**Pull request closed without merging**